multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2215,6 +2222,18 @@ def predict(
         if (as_pandas is None and isinstance(data, pd.DataFrame)) or as_pandas is True:
             pred = self._as_pandas(data=data, to_be_converted=pred)
 
+        if save_results:
+            ## Dumping Result for detection only now
+            if self._pipeline == OBJECT_DETECTION:
+                checkpoint_name = self._hyperparameters["model.mmdet_image.checkpoint_name"]
```

Comment:
this should be available in self._config or self._model

Reply:
oh okay. I wasn't aware the model name is already extracted.